velox/substrait/TypeUtils.cpp

```diff
@@ -43,6 +43,31 @@ int64_t bytesOfType(const TypePtr& type) {
   }
 }
 
+void getTypesFromCompoundName(
+    const std::string& compoundName,
+    std::vector<std::string>& types) {
```

Comment:
Any particular reason to use output parameter for types? It would be clear to use a return type:

```
std::vector<std::string> getTypesFromCompoundName(compoundName);
```

Reply:
> Any particular reason to use output parameter for types? It would be clear to use a return type:
> 
> ```
> std::vector<std::string> getTypesFromCompoundName(compoundName);
> ```

its fixed.